week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,14 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
 console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// - How many parameters does the function take? 2
+// - What is the function name? s
+// - Where is the function called? with what parameters? line 5
+// - What does the function return? code your
+// - How can this function be improved? add + on line 2 (return)
```

Comment:
Sorry, I don't think I understand - how would the code look like then?
Also, how about renaming the function and parameters? How much you name them?